Adds <p> tags to subscription and unsubscription success emails.
Fixed in 2.4.0.

Original issue: https://github.com/magento/magento2/issues/28166
PR: https://github.com/magento/magento2/pull/27357


diff --git a/view/frontend/email/subscr_success.html b/view/frontend/email/subscr_success.html
index d56163c..996dff0 100644
--- a/view/frontend/email/subscr_success.html
+++ b/view/frontend/email/subscr_success.html
@@ -13,6 +13,6 @@
 
 {{template config_path="design/email/header_template"}}
 
-{{trans "You have been successfully subscribed to our newsletter."}}
+<p>{{trans "You have been successfully subscribed to our newsletter."}}</p>
 
 {{template config_path="design/email/footer_template"}}
diff --git a/view/frontend/email/unsub_success.html b/view/frontend/email/unsub_success.html
index d39b5d8..1f222f8 100644
--- a/view/frontend/email/unsub_success.html
+++ b/view/frontend/email/unsub_success.html
@@ -13,6 +13,6 @@
 
 {{template config_path="design/email/header_template"}}
 
-{{trans "You have been unsubscribed from the newsletter."}}
+<p>{{trans "You have been unsubscribed from the newsletter."}}</p>
 
 {{template config_path="design/email/footer_template"}}
